docs/_includes/default.html

```diff
@@ -31,49 +31,102 @@ <h1>
-        />
-      </a>
-      <a
-        title="Mocha is an OpenJS Foundation Project"
-        href="https://openjsf.org"
-        rel="external noopener"
-        target="_blank"
-      >
-        <img
-          src="/images/openjsf-logo.svg"
-          width="300"
-          height="94"
-          alt="OpenJS Foundation Logo"
-        />
-      </a>
+      <div class="sponsorship">
+        <span>Services we consume</span>
+        <p class="image-list faded-images is-loaded">
+          <a href="https://matomo.org/" rel="external noopener" target="_blank">
+            <img
+              title="Matomo"
+              src="images/matomo-logo.png"
+              alt="Matomo Logo"
+            />
+          </a>
+          <a
+            href="https://travis-ci.com/"
+            rel="external noopener"
+            target="_blank"
+          >
+            <img
+              title="Travis CI"
+              src="images/travisCi-logo.png"
+              alt="Travis CI Logo"
+            />
+          </a>
+          <a
+            href="https://www.appveyor.com/"
+            rel="external noopener"
+            target="_blank"
+          >
+            <img
+              title="AppVeyor"
+              src="images/appVeyor-logo.png"
+              alt="AppVeyor Logo"
+            />
+          </a>
+          <a
+            href="https://www.netlify.com/"
+            rel="external noopener"
+            target="_blank"
+          >
+            <img
+              title="Netlify"
+              src="images/netlify-logo.png"
+              alt="Netlify Logo"
+            />
+          </a>
+          <a
+            href="https://coveralls.io"
+            rel="external noopener"
+            target="_blank"
+          >
+            <img
+              title="Coveralls"
+              src="images/coveralls-logo.png"
+              alt="Coveralls Logo"
+            />
+          </a>
+        </p>
+      </div>
     </aside>
 
     <footer>
-      <div>
-        <a rel="home" href="https://mochajs.org/">mochajs.org</a>
-        is licensed under a
-        <a
-          rel="license external noopener"
-          href="https://creativecommons.org/licenses/by/4.0/"
-        >
-          Creative Commons Attribution 4.0 International License</a
-        >.
-        <dl class="dl-inline last-modified">
-          <dt>Last updated</dt>
-          <dd>
-            <time
-              itemprop="lastModified"
-              datetime="{{ 'now' | date: '%Y-%m-%dT%H:%M:%SZ' }}"
-            >
-              {{ 'now' | date: '%a %b %d %H:%M:%S %Y' }}
-            </time>
-          </dd>
-        </dl>
+      <div class="footer-box">
+        <div class="footer-logo">
+          <a
+            title="Mocha is an OpenJS Foundation Project"
+            href="https://openjsf.org"
+            rel="external noopener"
+            target="_blank"
+          >
+            <img
+              src="images/openjsf-logo.png"
```

Comment:
Can't we use svg logo?

Reply:
I wanted to,
Here are the reasons I changed the file from svg to png.

1)  image resize problem
in Readme.md I wasn't able to use code below to resize file
`![](url path =250x250)
  ![](url path | width=100)
  ![](url path){ width: 200px; }
`
So if I want to resize file , all I have to do is to use <img />.
but in Readme, All contents are written in markdown language.
I don't want to break it because of my code.

2) file path problem
If I used img tag, I couldn't get the image using relative path.
like below
`
<img src='./docs/images/someimages.png'/> 
`
so I wanted to use file in docs/images/
`
https://raw.githubusercontent.com/mochajs/mocha/master/docs/images/matomo-logo.png
`
I had to use file path above. but as far as svg file is concerned, I wasn't able to get the right path to get the file.

so I just decided to make new file with the size I want. so the file turns out into png.
if i missed something, please let me know, thank you for your feed back.
I really appreciate it.